modules/cluster-logging-collecting-and-forwarding-logs-only.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-collector.adoc
+
+[id="cluster-logging-collecting-and-forwarding-logs-only_{context}"]
+= Collecting and forwarding logs only
+
+As an administrator, if are forwarding logs to only a third-party log store and not to the default Elasticsearch log store, you can remove several unused components from your logging cluster.
```

Comment:
>if are forwarding logs to only a third-party log store
if you are only forwarding logs to a third-party log store?

Reply:
In casual conversation, I use "only" the way you suggest. However, the meaning of "only" can become more ambiguous the further you move it from the "exclusive object." For technical writing, it should be as close to the "exclusive object" as possible.
 
 For example, "if you are only forwarding logs to a third-party log store," can take on an alternative meaning, such as "only forwarding logs and not doing something else to them."